### PR TITLE
Add yamllint for CI

### PR DESCRIPTION
**Pull request by bcongdon**
Validate the group store YAML as the CI step

### CODE REVIEW

File: .travis.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+language: python
+python:
+  - "2.7"
+script:
+  - "pip install yamllint && yamllint -c ./.yamllint.yaml *.yaml"
```
Comment (sameetandpotatoes) [inline .travis.yaml]: Why both `.travis.yaml` and `.travis.yml`?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,6 @@
 # groot-groups-store
+
+[![Build Status](https://travis-ci.org/acm-uiuc/groot-groups-store.svg?branch=master)](https://travis-ci.org/acm-uiuc/groot-groups-store)
+
+
```
Comment (sameetandpotatoes) [inline README.md]: NP: remove extra line

File: .yamllint.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+extends: default
+
+rules:
+  line-length: disable
```
Comment (sameetandpotatoes) [inline .yamllint.yaml]: Also, is there more that we can do? I saw this list of rules on some random repo, which may be a bit excessive, but some of them we might want to do:

```
rules:
  braces:
    max-spaces-inside: 1
  comments:
    level: error
  comments-indentation:
    level: warning
  document-end:
    present: no
  document-start:
    level: error
    present: no
  empty-lines:
    max: 1
    max-start: 0
    max-end: 0
  line-length:
    level: warning
    max: 120
```

File: .travis.yaml (hunk 1 above)
Comment (bcongdon) [inline .travis.yaml]: Good catch

File: .yamllint.yaml (hunk 3 above)
Comment (bcongdon) [inline .yamllint.yaml]: I tweaked this a bit, and added it. The linting is more strict now 👍

**Comment by narendasan** [comment]
@bcongdon @tommypacker  can we take this opportunity to verify the correct people are in each group?

**Comment by sameetandpotatoes** [comment]
@bcongdon the linting rules looks good now btw

File: committees.yaml [hunk 4]
```diff
@@ -2,64 +2,144 @@
...
+  members:
+    - Ann Rajan
+    - Min Park
+    - Vivek Vaidya
+    - Naren Dasan
+    - Eric Lee
+    - Sameet Sapra
+    - Caren Zeng
+    - Shivali Patel
+    - Tori Becú
```
Comment (tommypacker) [inline committees.yaml]: Remove Tori

File: committees.yaml [hunk 5]
```diff
@@ -2,64 +2,144 @@
...
+  netids:
+    - vkv2
+    - minhyuk2
+    - amrajan2
+    - sivagna2
+    - ericdl2
+    - ssapra2
+    - carenz2
+    - supatel2
+    - becu2
```
Comment (tommypacker) [inline committees.yaml]: Same as above

**Pull request merged**